data/json/npcs/exodii/exodii_merchant_itemlist.json

```diff
@@ -177,7 +177,8 @@
       [ "bio_scent_mask", 10 ],
       [ "bio_jointservo", 10 ],
       [ "bio_taste_blocker", 10 ],
-      [ "bio_trickle", 10 ]
+      [ "bio_trickle", 10 ],
+      [ "bio_ar", 10 ]
```

Comment:
You don't have to do so but I'd appreciate if you added these to the renfield CBM dissects in Xedra Evolved and the scavenger/pirate CBM dissect lists in Aftershock.